### ISSUE TITLE
Mix & Render commands should not include muted audio - mute should be honored

### ISSUE THREAD

**Issue by petersampsonaudacity** [audacity/audacity#7034]
### Your idea

Currently the Mix & Render commands (in the File > Mix menu) mix ALL selected tracks regardless of the mute staus of any of the selected tracks.

This is very inconsistent with Export and Playback where the muted tracks are not exported or played.

It would be better if the Mix and Render commands (and probably the Mix Stero Down to Mono command too) honored the mute settings in any selected tracks and did not mix from selected tracks.

### Use Case
The use case for this was a recent post on the Forum from user [bdbmkeeirsk11](https://forum.audacityteam.org/u/bdbmkeeirsk11)
This user had a multitrack project with some tracks muted and wanted to Mix after a select all and wanted the muted tracks to be excluded from the mix.

They have a workaround which is to export as WAV (which omits the mutes) and then import that mixed WAV track

See: https://forum.audacityteam.org/t/mix-and-render-to-new-track-would-like-it-to-omit-muted-tracks/113892

### Problem to be solved

Mix commands are inconsistent with Export and Playback wirg regard to muting

### Prior art

None

### Additional context

@SteveDaulton wrote in the Forum thread:

>I recall some years ago, this question being discussed at length by the developers. My position at that time was that “Mix and Render” should behave the same as mixdown on Export and ignore muted tracks. The opposing view is that Mix and Render should act on all selected tracks and ignore whether the track is muted or not, consistent with applying effects regardless of the track mute status. The latter argument won, mainly because that was how Audacity had always worked and long term users would be expecting Audacity to behave this way.

**Comment by Tantacrul** [comment]
Agreed 100%

Quick note. I'm finding it in Tracks > Mix, rather than File > Mix

**Comment by petersampsonaudacity** [comment]
> Agreed 100%
> 
> Quick note. I'm finding it in Tracks > Mix, rather than File > Mix

Thanks for that vote of support @Tantacrul 

And thanks for pointing out the menu error - I just corrected that.

**Comment by petersampsonaudacity** [comment]
BUMP

I note that @Tantacrul  gave 100% approval for this ten months ago on 5Auu24

But so far no progress seems to be happening on fixing this.  Are we likely to see this being fixed any time soon please?

**Comment by Tantacrul** [comment]
@grliszas14 - since we're working on export in AU4 right now, I propose we make sure this particular functionality is honoured.

Put simply - when rendering a mix from your Audacity project, if a track is muted, the render should exclude it.

@DilsonsPickles - probably worth including in the design as a note about rendering.

**Comment by Tantacrul** [comment]
Sorry, ignore the above. I thought this was a bug with existing export. This should be considered when we get to the 'Mix down to' feature in AU4 (currently in the 'Track' menu).

**Comment by petersampsonaudacity** [comment]
> I thought this was a bug with existing export.

@Tantacrul  -  this IS a long-standing bug with existing export in AU3.

Does your comment mean that it will not get fixed in AU3 only in AU4?

**Comment by Tantacrul** [comment]
@petersampsonaudacity - I just tried it in AU3.7.4. I had three tracks. Muted one and exported it as a .wav file. The result was a mix that didn't include the muted track. There's some confusion here methinks.

**Comment by petersampsonaudacity** [comment]
@Tantacrul  yes indeed Export has always honored mute status and only exported un-muted tracks.

Playback too honors mute status.

BUT the point of this bug that I posted is that **Mix&Render is inconsistent** in this regard in that it totally ignores mute status and mixes both un-muted and muted tracks.

**Comment by Tantacrul** [comment]
> _@Tantacrul - this IS a long-standing bug with existing export in AU3._

Then I'm not sure what you meant by this? Export is fine - which demotes this to a much lower priority, right?

**Comment by petersampsonaudacity** [comment]
> which demotes this to a much lower priority, right?

I guess so.

When all's said and done Mix only works on selected tracks (unlike Export which by default exports the whole project regardless of selection) - so if the user selects muted tracks as part of their mix then mix will include those selected tracks.

_So maybe this is a non-issue ..._
